src/renderer/components/FileDownloader.tsx

```diff
@@ -40,9 +54,18 @@ export const FileDownloader: FC<{ state: AppState }> = ({
     buttonText = 'PLAY';
   } else if (duringDownloadOrExtract) {
     className = 'DuringProcessButton';
-    buttonText = `${progress.toFixed(2)} %`;
+    if (process === Processes.download) {
+      buttonText = `${progress.toFixed(2)} %`;
+    } else {
+      buttonText = toShortSize(processingSize!);
+    }
     additionalInfo =
-      process === Processes.download ? 'Downloading...' : 'Extracting...';
+    // eslint-disable-next-line no-nested-ternary
+      process === Processes.download
+        ? 'Downloading...'
+        : process === Processes.extract
+        ? 'Extracting...'
+        : 'Error!!!!';
```

Comment:
nie obsługiwałbym errorów tutaj, zrobiłbym globalne podejście i zbudowałbym fallback screen, który zbierze info od usera co się stało + wyjaśni że będziemy pracowali nad naprawą tej funkjonalności.

Reply:
co do zasady zgoda, na razie chciałem by było widoczne kiedy jest błąd 
to jest dodatkowy stan aplikacji Process.error któy w designach trzeba jakoś obsłużyć narazie taka najprostsza obsługa

Reply:
ok zmienimy jak wejdzie sentry